packages/fast-components-react-base/src/number-field/number-field.schema.json

```diff
@@ -37,10 +37,6 @@
             "title": "HTML required attribute",
             "type": "boolean"
         },
-        "defaultValue": {
-            "title": "Initial value",
-            "type": "number"

```

Comment:
remove this or just type it as a string?

Reply:
I think we should remove for now.